src/renderer/base/renderer.cpp

```diff
@@ -91,7 +63,6 @@ IRenderData* Renderer::GetRenderData() const noexcept
         if (--tries == 0)
         {
             // Stop trying.
-            _pThread->DisablePainting();

```

Comment:
wha?

Reply:
It's way safer if this function simply signals the end of painting to the renderer. This reminds me that I didn't test this, shame on me.